lib/translator/translation.cc

```diff
@@ -87,15 +96,18 @@ ApiStatus Translation::Complete(Span<const nvme::GenericQueueEntryCpl> cpl_data,
 
   switch (opc) {
     case scsi::OpCode::kInquiry:
-      pipeline_status_ = InquiryToScsi(scsi_cmd_no_op, buffer, GetNvmeCmds());
+      pipeline_status_ =
+          InquiryToScsi(scsi_cmd_no_op, buffer_in, GetNvmeCmds());
       ret = ApiStatus::kSuccess;
       break;
+    case scsi::OpCode::kUnmap:
+      break;
```

Comment:
Do you not need to call a function here to set pipeline_status_ and check for errors from nvme?

Reply:
#150 handles errors independent of command logic. As of now pipeline_status_ doesn't do anything after this point in the logic (is set to invalid by AbortPipeline at the end of complete), but given all other cases have taken to setting it I'll add a setter here as well. No function is needed in general for unmap because there is no response data passed to the SCSI buffer.

Reply:
Was this addressed? I still don't see an assignment to pipeline_status_.

Reply:
We can set pipeline_status_ to success to be more explicit, but any errors stemming from the unmap command are handled upstream of this logic

Reply:
Makes sense, but we indeed should try to be locally consistent as well.